integration-tests/azure-servicebus/README.md

```diff
@@ -0,0 +1,196 @@
+# Azure Service Bus sample
+
+This is a sample about using the Quarkus extension to send/receive messages with Azure Service Bus.
+
+## Prerequisites
+
+To successfully run this sample, you need:
+
+* JDK 17+ installed with JAVA_HOME configured appropriately
+* Apache Maven 3.8.6+
+* Azure CLI and Azure subscription
+* Docker
+
+You also need to clone the repository and switch to the directory of the sample.
+
+```
+git clone https://github.com/quarkiverse/quarkus-azure-services.git
+cd quarkus-azure-services/integration-tests/azure-servicebus
+```
+
+### Use development iteration version
+
+By default, the sample is kept in sync with the development iteration version, which is `999-SNAPSHOT`. To install the development iteration version, build it locally.
+
+```
+mvn clean install -DskipTests --file ../../pom.xml
+```
+
+### Use release version
+
+If you want to use the release version, you need to update the version of dependencies in the `pom.xml` file.
+
+First, you need to find out the latest release version of the Quarkus Azure services extensions from [releases](https://github.com/quarkiverse/quarkus-azure-services/releases), for example, `1.1.4`.
+
+Then, update the version of dependencies in the `pom.xml` file, for example:
+
+```xml
+<parent>
+    <groupId>io.quarkiverse.azureservices</groupId>
+    <artifactId>quarkus-azure-services-parent</artifactId>
+    <version>1.1.4</version>
```

Comment:
@albers FYI - the version used here is just a `dummy` one as we haven't released the extension yet. The readme will be updated with the correct version during the release process.